bool isleftarm
int64 action
int64 target
---
bool success
